book/src/developers/protocols/json_rpc.md

```diff
@@ -32,13 +37,13 @@ The specification for these endpoints can be found [here](https://eth.wiki/json-
 - [`web3_clientVersion`](https://eth.wiki/json-rpc/API#web3_clientversion)
 
 ### Custom Trin JSON-RPC endpoints
-- [`portal_historyRadius`](#portal_historyRadius)
-- [`portal_stateRadius`](#portal_stateRadius)
-- [`portal_historyLocalContent`](#portal_historyLocalContent)
-- [`portal_stateLocalContent`](#portal_stateLocalContent)
-- [`portal_historyRecursiveFindContent`](#portal_historyRecursiveFindContent)
-- [`portal_historyTraceRecursiveFindContent`](#portal_historyTraceRecursiveFindContent)
-- [`portal_paginateLocalContentKeys`](#portal_paginateLocalContentKeys)
+The following endpoints are not part of the Portal Network specification and are defined
+in subsequent sections:
+- [`portal_historyRadius`](#portal_historyradius)
+- [`portal_historyRecursiveFindContent`](#portal_historyrecursivefindcontent)
```

Comment:
`portal_*RecursiveFindContent` is part of the [specs](https://playground.open-rpc.org/?schemaUrl=https://raw.githubusercontent.com/ethereum/portal-network-specs/assembled-spec/jsonrpc/openrpc.json&uiSchema%5BappBar%5D%5Bui:splitView%5D=false&uiSchema%5BappBar%5D%5Bui:input%5D=false&uiSchema%5BappBar%5D%5Bui:examplesDropdown%5D=false). Let's move them in the list above.

Reply:
Ok `historyRecursiveFindContent` is moved now, but `portal_historyTraceRecursiveFindContent` remains as I think that is Trin-specific.